books/RayTracingTheRestOfYourLife.html

```diff
@@ -1102,31 +1128,42 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
         }
         std::cout << std::fixed << std::setprecision(12);
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
         std::cout << "I = " << sum/N << '\n';
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
     }
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [integ-xsq-2]: <kbd>[integrate_x_sq.cc]</kbd> Integrating $x^2$ with PDF]
-</div>
+    [Listing [integ-xsq-2]: <kbd>[integrate_x_sq.cc]</kbd> Explicit uniform PDF for $x^2$]
 
 
-Importance Sampling
---------------------
-Since we are sampling more where the PDF is big, we might expect less noise and thus faster
-convergence. In effect, we are steering our samples toward the parts of the distribution that are
-more _important_. This is why using a carefully chosen nonuniform PDF is usually called _importance
-sampling_.
+There are a couple of important things to emphasize. Every value of $x$ represents one sample of the
+function $x^2$ within the distribution [0,2]. We use a function $f$ to randomly select samples from
+within this distribution. We are no longer multiplying the output sum by $2$ to arrive at the final answer. We were previously multiplying the average over the interval (sum/N) times the length of the interval ($b - a = 2$). Here, we don't need to multiply by the interval length.
+
+
+We need to account for the nonuniformity of the PDF of $x$. Failing to account for this
+nonuniformity will introduce bias in our scene, indeed this bias is the source of our inaccurately
+bright image, so if we account for nonuniformity, we will get accurate results. The PDF will "steer"
+samples toward specific parts of the distribution, which will cause us to converge faster, but this
+will introduce bias. To remove bias we need to down-weight where we sample more frequently, and
```

Comment:
“… converge faster, but at the cost of introducing bias. To remove this bias, we need …”